docs/content/guides/cell-types/numeric-cell-type.md

```diff
@@ -11,10 +11,14 @@ canonicalUrl: /numeric-cell-type
 
 ## Overview
 
-By default, Handsontable treats all cell values as `string` type. This is because the `<textarea>` returns a string as its value. There are many cases where you need cell values to be treated as a `number` type. The numeric cell type allows you to format displayed numbers nicely and sort them correctly.
+By default, Handsontable treats all cell values as `string` type. This is because the text editor returns a string as its value. There are many cases where you need cell values to be treated as a `number` type. The numeric cell type allows you to format displayed numbers nicely and sort them correctly.
```

Comment:
I would not remove this technical detail.

Reply:
Why not? My reasoning is: 

1. The old text uses the HTML element `<textarea>` without context, which is not very informative. It was never introduced what `<textarea>` has to do with the numeric cell type.
2. `<textarea>`is used by the text cell editor. The numeric cell editor only uses it through inheritance (like almost all other editors). If it is so implementation detail is so important, why should it be given just on this particular page? 

I agree that this change is not really in scope of this PR, so if the above explanation does not convince you, I will revert it.

Reply:
1. I can agree with that.
2. That's how Handsontable works for now. I think that new text explain how Handsontable works, but without implementation details it may sound strange - like our whim to do some kind of casting. That's not true, the change comes from the idea of reading from DOM element. It's documentation for developers, so I would leave it.

Maybe this description isn't perfect, but for me it have more information. I would leave it as is.

Reply:
The more I think about it, the more I find the old the explanation not satisfying.

> By default, Handsontable treats all cell values as `string` type. This is because the `<textarea>` returns a string as its value.

That was an explanation in 2012, when much of Handsontable's behavior relied directly on DOM. But currently, the behavior is dictated by conscious choices, not by underlying DOM elements.

May I propose a compromise: 1fd2ad6b36d0ed8943f29e179bac1907a5cfa857. WDYT?